release_notes/ocp_3_7_release_notes.adoc

```diff
@@ -0,0 +1,1534 @@
+
+- When `accessTokenMaxAgeSeconds` is set to `0`, tokens do not expire.
+- When left blank, {product-title} uses the definition in `master-config`.
+- Edit the client of interest via:
++
+----
+# oc edit oauthclients openshift-browser-client
+----
+
+- Set `accessTokenMaxAgeSeconds` to `600`.
+- Check the setting via:
++
+----
+# oc get oauthaccesstoken
+----
+
+See
+xref:../architecture/additional_concepts/other_api_objects.adoc#accessTokenMaxAgeSeconds[Other
+API Objects] for more information.
+
+[[ocp-37-storage]]
+=== Storage
+
+[[ocp-37-local-persistent-volumes]]
+==== Local Storage Persistent Volumes (Technology Preview)
+
+Local storage persistent volumes is currently in
+xref:ocp-37-technology-preview[Technology Preview].
+
+Local persistent volumes (PVs) now offer the ability to allow tenants to request
+storage that is local to a node through the regular persistent volume claim
+(PVC) process without needing to know the node.  Local storage is commonly used
+in data store applications.
+
+The administrator needs to create the local storage on the nodes, mount them
+under directories, and then manually create the persistent volume (PV).
+Alternatively, they can use an external provisioner and feed it the node
+configuration via `configMaps`.
+
+Example persistent volume named `example-local-pv` that some tenant can now claim:
+
+----
+apiVersion: v1
+kind: PersistentVolume
+metadata:
+  name: example-local-pv
+  annotations:
+    "volume.alpha.kubernetes.io/node-affinity": '{
+      "requiredDuringSchedulingIgnoredDuringExecution": {
+        "nodeSelectorTerms": [
+          { "matchExpressions": [
+            { "key": "kubernetes.io/hostname",
+              "operator": "In",
+              "values": ["my-node"]
+            }
+          ]}
+         ]}
+        }'
+spec:
+  capacity:
+    storage: 5Gi
+  accessModes:
+  - ReadWriteOnce
+  persistentVolumeReclaimPolicy: Retain
+  storageClassName: local-storage
+  local:
+    path: /mnt/disks/vol1
+----
+
+See
+xref:../install_config/configuring_local.adoc#install-config-configuring-local[Configuring
+for Local Volume] and
+xref:../install_config/persistent_storage/persistent_storage_local.adoc#install-config-persistent-storage-persistent-storage-local[Persistent
+Storage Using Local Volume] for more information.
+
+[[ocp-37-tenant-driven-storage-snapshotting]]
+==== Tenant-driven Storage Snapshotting (Technology Preview)
+
+Tenant-driven storage snapshotting is currently in
+xref:ocp-37-technology-preview[Technology Preview].
+
+Tenants now have the ability to leverage the underlying storage technology
+backing the persistent volume (PV) assigned to them to make a snapshot of their
+application data. Tenants can also now restore a given snapshot from the past to
+their current application.
+
+An external provisioner is used to access the EBS, GCE pDisk, HostPath, and
+Cinder snapshotting API. This Technology Preview feature has tested EBS and
```

Comment:
As far as I can see, Cinder is not supported at the moment. https://trello.com/c/owCRnP8I/521-13-snapshot-implement-the-out-of-tree-snapshotting-controller

Reply:
@xltian what test cases do we have in this area?